jvm/build.sbt

```diff
@@ -18,6 +18,9 @@ val graalvmJsVersion = "24.0.1"
 val coursierVersion = "2.1.10"
 val gatlingVersion = "3.11.4"
 
+// bit weird cause this is not a dependency of this project
+val gatlingEnterpriseComponentPluginVersion = "1.9.2-SNAPSHOT"
```

Comment:
Waiting for a release

Reply:
The current latest release is 1.9.3 ;)
We can release whenever you want.